src/components/Footer.tsx

```diff
@@ -0,0 +1,84 @@
+import Image from "next/image";
+import Link from "next/link";
+
+function List({ text, children }: { text: string; children: React.ReactNode }) {
+  return (
+    <div>
+      <h2 className="mb-5 text-base font-medium">{text}</h2>
+      <ul>{children}</ul>
+    </div>
+  );
+}
+
+function ListItem({
+  url,
+  target = "_blank",
+  text,
+}: {
+  url: string;
+  target?: string;
+  text: string;
+}) {
+  return (
+    <li className="mb-2 text-sm">
+      <Link href={url} target={target}>
+        {text}
+      </Link>
+    </li>
+  );
+}
+
+function Footer() {
+  return (
+    <>
+      <div className="grid place-items-center md:mb-16 md:mt-24">
+        <Image
+          src="/Juwe2025.png"
+          alt="Description of the image"
```

Comment:
Co to jest za alt 😭

Reply:
to był defaultowy z next/image i przeoczyłem go 😅

Reply:
jak coś to w najnowszym commicie nie zmieniłeś alta

Reply:
aa dobra bo jeden naprawiłem a jeszcze drugi był XD